content/release-10-16-0/applications-10-16-0-bundle/10-16-0-0.md

```diff
@@ -187,6 +187,17 @@ MTM-47676</td>
 <td>MTM-47517</td>
 </tr>
 
+<tr>
+<td>Cockpit</td>
+<td>The "Welcome" widget in the Cockpit application is now reliably displaying the <b>Register device</b> link as long as the "Devicemanagement" application is subscribed to the tenant.</td>
+<td>MTM-49324</td>
+</tr>
+
+<tr>
+<td>Java SDK</td>
+<td>Fixed an issue with the AlarmRepresentation comparison in equals and hashcode methods. Now, instead of comparing the full ManagedObjectRepresentation as source of alarms, we only compare the managed objects ID. There is no change in ManagedObjectRepresentation.</td>

```

Comment:
```suggestion
<td>Fixed an issue with the AlarmRepresentation comparison in equals and hashcode methods. Now, instead of comparing the full ManagedObjectRepresentation as the source of alarms, we only compare the managed objects ID. There is no change in ManagedObjectRepresentation.</td>
```
there could also be < code > formatting on AlarmRepresentation etc.